sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -12,6 +12,14 @@
 #include "azure/core/http/http.hpp"
 #include "azure/core/http/policies/policy.hpp"
 #include "azure/core/http/transport.hpp"
+#include "azure/core/platform.hpp"
+
+#if defined(AZ_PLATFORM_LINUX)
+#include <openssl/opensslv.h>
+#if OPENSSL_VERSION_NUMBER >= 0x00905100L
+#define _azure_SUPPORT_SETTING_CAPATH
+#endif // OPENSSL_VERSION_NUMBER >= 0x00905100L
```

Comment:
I am not sure we need this openssl version check.

0x00905100L seems to correspond to 0.9.5 which is 20+ years old. We can assume the version of openssl is 1.1.1 or higher, since we don't support older versions anyway.

Reply:
Which means we can remove the `_azure_SUPPORT_SETTING_CAPATH` and assume this option is available wherever `(AZ_PLATFORM_LINUX)` is defined.